gitlab-ci: external connections issues

Tarantool version:
All

OS version:
All

Bug description:
1. OSX 15 only on mac4
https://gitlab.com/tarantool/tarantool/-/jobs/835453933
```
$ /bin/bash -c "git fetch -p && git checkout -f ${CI_COMMIT_SHORT_SHA}"
fatal: «https://gitlab.com/tarantool/tarantool.git/» недоступно: Could not resolve host: gitlab.com
```

2. On Linux for CentOS *
https://gitlab.com/tarantool/tarantool/-/jobs/835799341
```
Extra Packages for Enterprise Linux 8 - x86_64  7.4 kB/s | 3.4 kB     00:00    
Failed to download metadata for repo 'epel'
Error: Failed to download metadata for repo 'epel'
Extra Packages for Enterprise Linux 8 - x86_64  7.4 kB/s | 3.4 kB     00:00    
Failed to download metadata for repo 'epel'
Error: Failed to download metadata for repo 'epel'
make: *** [/pack//rpm.mk:162: package] Error 1
```

3. On Linux for Fedora *
https://github.com/tarantool/tarantool/runs/1933471340
```
Fedora Modular 29 - x86_64 - Updates            0.0  B/s |   0  B     00:29    
Failed to synchronize cache for repo 'updates-modular'
Error: Failed to synchronize cache for repo 'updates-modular'
make: *** [/pack//rpm.mk:162: package] Error 1
```

--- a/centos/7/Dockerfile
+++ b/centos/7/Dockerfile
@@ -6,6 +6,10 @@ ARG DEVTOOLSET=devtoolset-8
 # Fix missing locales
 ENV LC_ALL="en_US.UTF-8" LANG="en_US.UTF-8"
 
+# Update repositories and installed packages to avoid of issues got at:
+#   https://github.com/tarantool/tarantool-qa/issues/60
+RUN yum update -v -y
+
 # Enable extra repositories
 RUN yum -y install \
     wget \
@@ -65,3 +69,6 @@ RUN sed -e '/\[updates-source\]/,+6d' -i /etc/yum.repos.d/CentOS-Sources.repo
 
 # The same as above, but for the base-source repository.
 RUN sed -e '/\[base-source\]/,+6d' -i /etc/yum.repos.d/CentOS-Sources.repo
+
+# Cleanup YUM metadata and decrease image size
+RUN yum clean all

--- a/centos/8/Dockerfile
+++ b/centos/8/Dockerfile
@@ -1,6 +1,10 @@
 FROM centos:8
 MAINTAINER Alexander V. Tikhonov <avtikhon@tarantool.org>
 
+# Update repositories and installed packages to avoid of issues got at:
+#   https://github.com/tarantool/tarantool-qa/issues/60
+RUN dnf update -v -y
+
 # Enable extra tools
 RUN yum -y install wget yum-utils
 
@@ -25,3 +29,6 @@ RUN yum -y install \
 
 # Enable sudo without tty
 RUN sed -i.bak -n -e '/^Defaults.*requiretty/ { s/^/# /;};/^%wheel.*ALL$/ { s/^/# / ;} ;/^#.*wheel.*NOPASSWD/ { s/^#[ ]*//;};p' /etc/sudoers
+
+# Cleanup DNF metadata and decrease image size
+RUN dnf clean all

--- a/fedora/28/Dockerfile
+++ b/fedora/28/Dockerfile
@@ -4,6 +4,10 @@ MAINTAINER Alexey Kopytov <akopytov@gmail.com>
 # Fix missing locales
 ENV LC_ALL="C.UTF-8" LANG="C.UTF-8"
 
+# Update repositories and installed packages to avoid of issues got at:
+#   https://github.com/tarantool/tarantool-qa/issues/60
+RUN dnf update -v -y
+
 # Install base toolset
 RUN dnf -y group install 'Development Tools'
 RUN dnf -y group install 'C Development Tools and Libraries'
@@ -16,3 +20,6 @@ ENV PATH /usr/lib/ccache:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/bin:
 
 # Enable sudo without tty
 RUN sed -i.bak -n -e '/^Defaults.*requiretty/ { s/^/# /;};/^%wheel.*ALL$/ { s/^/# / ;} ;/^#.*wheel.*NOPASSWD/ { s/^#[ ]*//;};p' /etc/sudoers
+
+# Cleanup DNF metadata and decrease image size
+RUN dnf clean all

--- a/fedora/29/Dockerfile
+++ b/fedora/29/Dockerfile
@@ -4,6 +4,12 @@ MAINTAINER Alexey Kopytov <akopytov@gmail.com>
 # Fix missing locales
 ENV LC_ALL="C.UTF-8" LANG="C.UTF-8"
 
+# Update repositories and installed packages to avoid of issues got at:
+#   https://github.com/tarantool/tarantool-qa/issues/60
+RUN echo "Remove disabled 'failovermethod' since Fedora 29"
+RUN sed -i '/^failovermethod=/d' /etc/yum.repos.d/*.repo
+RUN dnf update -v -y
+
 # Install base toolset
 RUN dnf -y group install 'Development Tools'
 RUN dnf -y group install 'C Development Tools and Libraries'
@@ -16,3 +22,6 @@ ENV PATH /usr/lib/ccache:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/bin:
 
 # Enable sudo without tty
 RUN sed -i.bak -n -e '/^Defaults.*requiretty/ { s/^/# /;};/^%wheel.*ALL$/ { s/^/# / ;} ;/^#.*wheel.*NOPASSWD/ { s/^#[ ]*//;};p' /etc/sudoers
+
+# Cleanup DNF metadata and decrease image size
+RUN dnf clean all

--- a/fedora/30/Dockerfile
+++ b/fedora/30/Dockerfile
@@ -4,6 +4,12 @@ MAINTAINER Alexander Turenko <alexander.turenko@tarantool.org>
 # Fix missing locales
 ENV LC_ALL="C.UTF-8" LANG="C.UTF-8"
 
+# Update repositories and installed packages to avoid of issues got at:
+#   https://github.com/tarantool/tarantool-qa/issues/60
+RUN echo "Remove disabled 'failovermethod' since Fedora 29"
+RUN sed -i '/^failovermethod=/d' /etc/yum.repos.d/*.repo
+RUN dnf update -v -y
+
 # Install base toolset
 RUN dnf -y group install 'Development Tools'
 RUN dnf -y group install 'C Development Tools and Libraries'
@@ -16,3 +22,6 @@ ENV PATH /usr/lib/ccache:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/bin:
 
 # Enable sudo without tty
 RUN sed -i.bak -n -e '/^Defaults.*requiretty/ { s/^/# /;};/^%wheel.*ALL$/ { s/^/# / ;} ;/^#.*wheel.*NOPASSWD/ { s/^#[ ]*//;};p' /etc/sudoers
+
+# Cleanup DNF metadata and decrease image size
+RUN dnf clean all

--- a/fedora/31/Dockerfile
+++ b/fedora/31/Dockerfile
@@ -4,6 +4,10 @@ MAINTAINER Alexey Kopytov <akopytov@gmail.com>
 # Fix missing locales
 ENV LC_ALL="C.UTF-8" LANG="C.UTF-8"
 
+# Update repositories and installed packages to avoid of issues got at:
+#   https://github.com/tarantool/tarantool-qa/issues/60
+RUN dnf update -v -y
+
 # Install base toolset
 RUN dnf -y group install 'Development Tools'
 RUN dnf -y group install 'C Development Tools and Libraries'
@@ -19,3 +23,6 @@ ENV PATH /usr/lib/ccache:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/bin:
 
 # Enable sudo without tty
 RUN sed -i.bak -n -e '/^Defaults.*requiretty/ { s/^/# /;};/^%wheel.*ALL$/ { s/^/# / ;} ;/^#.*wheel.*NOPASSWD/ { s/^#[ ]*//;};p' /etc/sudoers
+
+# Cleanup DNF metadata and decrease image size
+RUN dnf clean all

--- a/fedora/32/Dockerfile
+++ b/fedora/32/Dockerfile
@@ -4,6 +4,10 @@ MAINTAINER Alexander V. Tikhonov <avtikhon@tarantool.org>
 # Fix missing locales
 ENV LC_ALL="C.UTF-8" LANG="C.UTF-8"
 
+# Update repositories and installed packages to avoid of issues got at:
+#   https://github.com/tarantool/tarantool-qa/issues/60
+RUN dnf update -v -y
+
 # Install base toolset
 RUN dnf -y group install 'Development Tools'
 RUN dnf -y group install 'C Development Tools and Libraries'
@@ -19,3 +23,6 @@ ENV PATH /usr/lib/ccache:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/bin:
 
 # Enable sudo without tty
 RUN sed -i.bak -n -e '/^Defaults.*requiretty/ { s/^/# /;};/^%wheel.*ALL$/ { s/^/# / ;} ;/^#.*wheel.*NOPASSWD/ { s/^#[ ]*//;};p' /etc/sudoers
+
+# Cleanup DNF metadata and decrease image size
+RUN dnf clean all

--- a/fedora/33/Dockerfile
+++ b/fedora/33/Dockerfile
@@ -4,6 +4,10 @@ MAINTAINER Alexander V. Tikhonov <avtikhon@tarantool.org>
 # Fix missing locales
 ENV LC_ALL="C.UTF-8" LANG="C.UTF-8"
 
+# Update repositories and installed packages to avoid of issues got at:
+#   https://github.com/tarantool/tarantool-qa/issues/60
+RUN dnf update -v -y
+
 # Install base toolset
 RUN dnf -y group install 'Development Tools'
 RUN dnf -y group install 'C Development Tools and Libraries'
@@ -16,3 +20,6 @@ ENV PATH /usr/lib/ccache:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/bin:
 
 # Enable sudo without tty
 RUN sed -i.bak -n -e '/^Defaults.*requiretty/ { s/^/# /;};/^%wheel.*ALL$/ { s/^/# / ;} ;/^#.*wheel.*NOPASSWD/ { s/^#[ ]*//;};p' /etc/sudoers
+
+# Cleanup DNF metadata and decrease image size
+RUN dnf clean all

--- a/fedora/rawhide/Dockerfile
+++ b/fedora/rawhide/Dockerfile
@@ -4,6 +4,10 @@ MAINTAINER Roman Tsisyk <roman@tarantool.org>
 # Fix missing locales
 ENV LC_ALL="C.UTF-8" LANG="C.UTF-8"
 
+# Update repositories and installed packages to avoid of issues got at:
+#   https://github.com/tarantool/tarantool-qa/issues/60
+RUN dnf update -v -y
+
 # Install base toolset
 RUN dnf -y group install 'Development Tools'
 RUN dnf -y group install 'C Development Tools and Libraries'
@@ -23,3 +27,6 @@ ENV PATH /usr/lib/ccache:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/sbin:/bin:
 
 # Enable sudo without tty
 RUN sed -i.bak -n -e '/^Defaults.*requiretty/ { s/^/# /;};/^%wheel.*ALL$/ { s/^/# / ;} ;/^#.*wheel.*NOPASSWD/ { s/^#[ ]*//;};p' /etc/sudoers
+
+# Cleanup DNF metadata and decrease image size
+RUN dnf clean all

